conf/messages.cy

```diff
@@ -366,6 +366,8 @@ a11yStatement.somePeople = Mae’n bosibl y bydd rhai pobl yn cael anhawster i d
 a11yStatement.difficultyList.1 = Ar y tudalennau ‘Hanes taliadau’ a ‘Ffurflenni TAW blaenorol’, pan fo’r tabiau’n cael eu clicio, nid yw’r defnyddiwr yn cael gwybod bod y dudalen wedi’i hadnewyddu
 a11yStatement.difficultyList.2 = Ar y tudalennau ‘Hanes taliadau’ a ‘Ffurflenni TAW blaenorol’, pan fo’r defnyddiwr yn clicio ar dab, mae ffocws y dudalen yn mynd at frig y dudalen
 a11yStatement.reportingProblems = Rhoi gwybod am broblemau hygyrchedd gyda’r gwasanaeth hwn
+a11yStatement.reportProblem.1 = Rydym bob amser yn ceisio gwella hygyrchedd y gwasanaeth hwn. Os byddwch yn dod o hyd i unrhyw broblem nad yw wedi’i rhestru ar y dudalen hon, neu os ydych o’r farn nad ydym yn bodloni gofynion hygyrchedd,
+a11yStatement.reportProblem.2 = rhowch wybod am y broblem hygyrchedd
 a11yStatement.lookingToImprove.1 = Rydym bob amser yn ceisio gwella hygyrchedd y gwasanaeth hwn. Os byddwch yn dod o hyd i unrhyw broblemau nad ydynt wedi’u rhestru ar y dudalen hon, neu os ydych o’r farn nad ydym yn bodloni gofynion hygyrchedd, cysylltwch â
```

Comment:
Could you please remove the Welsh messages for `allyStatement.lookingToImprove.1` and `allyStatement.lookingToImprove.2` as they are no longer used.